jira/resources.py

```diff
@@ -763,14 +764,46 @@ def __init__(
             self._parse_raw(raw)
         self.raw: Dict[str, Any] = cast(Dict[str, Any], self.raw)
 
-    def update(self, fields=None, async_=None, jira=None, body="", visibility=None):
-        """Update a comment"""
-        data = {}
+    # mypy complains about the signature as the order of the parameters is different.
+    # If we follow mypy here we will break backwards compatibility with version <= 3.2 which is probably not advised
+    @typing.no_type_check
+    def update(
```

Comment:
I believe the correct way for this case is to [`# type: ignore[override]`](https://mypy.readthedocs.io/en/stable/common_issues.html?#incompatible-overrides) - which will still type check the rest of the funciton but indicate to mypy this is a intentional break from the super class

Reply:
I'll fix this!

Reply:
didnt work :-S

Reply:
ok worked out if i move it to the first line of update